net-analyzer/ossec-hids/ossec-hids-3.2.0.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user
+
+DESCRIPTION="Open Source Host-based Intrusion Detection System"
+HOMEPAGE="https://www.ossec.net/"
+SRC_URI="https://github.com/ossec/ossec-hids/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="agent hybrid local mysql postgres server sqlite"
+REQUIRED_USE="^^ ( agent hybrid local server )
+	?? ( mysql postgres )"
+
+DEPEND="mysql? ( virtual/mysql )
+	sqlite? ( dev-db/sqlite:3 )
+	postgres? ( dev-db/postgresql:= )
+	sys-libs/zlib:="
```

Comment:
zlib sub-slot binding shouldn't be used.

Reply:
I am using this because Michał Górny recommended doing so: (https://github.com/gentoo/gentoo/pull/10189#discussion_r226864835).

Reply:
Well, looks like @mgorny can make some mistakes, then ;)

The zlib soname won't change so the subslot binding is wrong. Also what @Zlogene said then should be applied now, and this zlib line removed since it's part of system set anyway and always present.

Reply:
I removed the line.